npm/react/examples/webpack-file/cypress.config.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
   'video': false,
   'fixturesFolder': false,
-  'testFiles': '**/*cy-spec.js',
   'viewportWidth': 500,
   'viewportHeight': 500,
   'component': {
+    'specPattern': '**/*cy-spec.js',
```

Comment:
Should this be cypress/component/**/*cy-spec.js?  Seems like we did that in npm/react/examples/webpack-options/cypress.config.js

Reply:
Yep, fixed https://github.com/cypress-io/cypress/pull/19319/commits/dc67b143ed92ac72a306d437dc83a236aa69fb81#diff-d9606627b12888cd15239859eb4597600250d590fafbf51236413a5633a89b52L7